scripts/maintenance/computational-clusters/osparc_clusters.py

```diff
@@ -29,6 +30,8 @@
 
 _SSH_USER_NAME: Final[str] = "ubuntu"
 
+_TaskCancelEventName = "cancel_event_{}"
```

Comment:
TIP: strings with format symbols, i.e. `{}`, can have a `NameTemplate` suffix to distinguish from constants

```python
_TaskCancelEventNameTemplate: Final[str] = `cancel_event_{}`
```

Reply:
done